_pages/faq/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Frequently Asked Questions
+---
+
+### Q: Does there exist an exhaustive/comprehensive/authoritative list of tools that are available for use?  or a list of what tools are prohibited?
+
+A:  [https://ea.gsa.gov](https://ea.gsa.gov/)
+
+### Q: What is the definition of a tool?  Examples of tools on the TTS Handbook include SaaS solutions (e.g., Smartsheet) as well as applications installed on a system that don't include a dependency on network access (e.g., vim), so I don't think FedRAMP status (or lack thereof) is a determining factor.
```

Comment:
Instead of an FAQ, I think it would make more sense to just [define "tools" in context](https://handbook.tts.gsa.gov/#tools).

Reply:
hmmm ... curious ... 

Are you opposed to having a FAQ in general, or just having these listed as FAQs?

I think the benefit of having certain questions, especially these ones, is that in my few years at TTS, I feel as though this is something that has been asked, re asked, asked again, then asked some more (I can dig up examples if necessary). 

But I can see that these FAQs might be temporary until the work for the next steps for the Improve [SaaS Authorization documentation](https://docs.google.com/presentation/d/1gII0F9PtFvFWS1Pwqk5U6fT4XI9O7DWg6EatxTgMi8U/edit#slide=id.g8e60e68636_0_927) happen

Reply:
related [slack conversation ](https://gsa-tts.slack.com/archives/C0DCAMLQH/p1631309746020800)